sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqlCompiler/compiler/frontend/statements/CreateRelationStatement.java

```diff
@@ -132,6 +132,7 @@ public JsonNode getDefinedObjectSchema() {
         for (RelColumnMetadata col: this.columns) {
             ObjectNode column = fields.addObject();
             column.put("name", col.getName());
+            column.put("case_sensitive", col.nameIsQuoted);
```

Comment:
Will this work correctly for outputs?

Reply:
Let me add a test where the users explicitly specify the column names in the view. We don't have any of these yet.
I think the question only makes sense for that case.
And the answer is probably "no".

Reply:
> I think the question only makes sense for that case.

Right, but also we want to make sure to output `case_sensitive=false` for columns with auto-generated names.